src/index.js

```diff
@@ -18,7 +23,159 @@ import { months } from './data.js'
   leftArrow.addEventListener('click', () => { scrollItems('left') })
   rightArrow.addEventListener('click', () => { scrollItems('right') })
   document.getElementById('select-icon').addEventListener('change', updateMiddleCitiesScrollButtonVisibility)
+  changeBottomByContinents('down-arrow')
+  changeBottomByTemperature('up-arrow-temp')
+  document.querySelector('.continent-arrow').addEventListener('click', (e) => { changeBottomByContinents(e.target.id) })
+  document.querySelector('.temperature').addEventListener('click', (e) => { changeBottomByTemperature(e.target.id) })
+
+  /**
+   *  This function will update the bottom container elements based on continent sorting criteria
+   * @param {string} arrowId It is the arrow id of the arrow image
+   */
+  function changeBottomByContinents (arrowId) {
+    const continentArrow = document.getElementsByClassName('continent-arrow')[0]
+    if (arrowId === 'down-arrow') {
+      bottomItemCount = 0
```

Comment:
Few expressions are done in both the cases and can be moved out of the if else cases